diff --git a/data/org.gnome.gnome-flashback.gschema.xml.in.in b/data/org.gnome.gnome-flashback.gschema.xml.in.in
index 5c766a2..262409a 100644
--- a/data/org.gnome.gnome-flashback.gschema.xml.in.in
+++ b/data/org.gnome.gnome-flashback.gschema.xml.in.in
@@ -41,4 +41,14 @@
 			<_description>If set to true, then fade effect will be used to change the desktop background.</_description>
 		</key>
 	</schema>
+
+	<schema id="org.gnome.gnome-flashback.overrides" path="/org/gnome/gnome-flashback/overrides/">
+		<key name="button-layout" type="s">
+			<default>"appmenu:minimize,maximize,close"</default>
+			<_summary>Arrangement of buttons on the titlebar</_summary>
+			<_description>
+			  This key overrides the key in org.gnome.desktop.wm.preferences when running GNOME Flashback.
+			</_description>
+		</key>
+	</schema>
 </schemalist>
